Solutions/Marain.TenantManagement.Specs/Features/TenantManagement/CreateServiceTenant.feature

```diff
@@ -0,0 +1,40 @@
+﻿@perScenarioContainer
+@useInMemoryTenantProvider
+
+Feature: Adding a new Service tenant
+	In order to enable a Marain service for enrollment
+	As an administrator
+	I want to create a new service tenant for the service
+
+Background:
+	Given I have a service manifest called 'Operations Manifest' for a service called 'Operations v1'
+	Given I have a service manifest called 'Workflow Manifest' for a service called 'Workflow v1'
+	And the service manifest called 'Workflow Manifest' has the following dependencies
+	| Service Name |
+	| Operations   |
```

Comment:
I'm confused. I was expecting `Operations v1` here.

Reply:
Yes, I can see why that would be confusing. I've tweaked the tests to make it more obvious what's going on.